### PR TITLE
Upgrade Kubernetes to v1.18.6

### PR DESCRIPTION
**Pull request by to-bar**

### CODE REVIEW

**Comment by przemyslavic** [comment]
All possible configurations were tested to deploy clusters, deploy HA clusters, deploy on a single machine, upgrade from 0.4.4, 0.5.4, 0.6.0, 0.6.0 HA, promote to HA, offline installation.

File: core/src/epicli/data/common/ansible/playbooks/upgrade.yml [hunk 1]
```diff
@@ -59,7 +70,7 @@
     - import_role:
         name: upgrade
         tasks_from: kubernetes
-      vars: { ver: "1.17.7", cni_ver: "0.8.6", cni_in_kubelet: true }
+      vars: { ver: "1.18.6", cni_ver: "0.8.6", last_upgrade: true }
```
[the remark this answers is not in the document]
Comment (sk4zuzu) [reply core/src/epicli/data/common/ansible/playbooks/upgrade.yml:73]: @to-bar maybe something like `upgrade_final_step` or `upgrade_is_final_step` ? 🤔

[the remark this answers is not in the document]
Comment (rafzei) [reply core/src/epicli/data/common/ansible/playbooks/upgrade.yml:73]: hmm I think its quite clear variable name and means that it's a 'last upgrade run'. If the variable is set than some tasks should be performed. @mkyc do you have a better idea how to call it?

[the remark this answers is not in the document]
Comment (mkyc) [reply core/src/epicli/data/common/ansible/playbooks/upgrade.yml:73]: is_last_upgrade_run :)

[the remark this answers is not in the document]
Comment (to-bar) [reply core/src/epicli/data/common/ansible/playbooks/upgrade.yml:73]: > 
> 
> can we have better name for that variable `lat_upgrade`? It has no meaning.

Do you think `final_upgrade` is better or `upgrade_to_final_version` / `upgrade_to_target_version`? There may be a few upgrades (e.g. first 1.14 -> 1.15,  second 1.15 -> 1.16, ..., last 1.17 -> 1.18) and I need a flag to run some tasks once (during the last upgrade).

[the remark this answers is not in the document]
Comment (mkyc) [reply core/src/epicli/data/common/ansible/playbooks/upgrade.yml:73]: `upgrade_to_final_version` seams most informative from propositions.

**Pull request merged**